src/ontology/mondo-edit.obo

```diff
@@ -230445,9 +230445,11 @@ xref: OMIM:606215 {source="MONDO:equivalentTo"}
 xref: Orphanet:98722 {source="OMIM:606215"}
 xref: UMLS:C1389016 {source="OMIM:606215", source="MONDO:notFoundInDiseaseSubset"}
 xref: UMLS:C1853513 {source="OMIM:606215", source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset"}
-is_a: MONDO:0003847 ! Mendelian disease
-is_a: MONDO:0020290 {source="DC-OMIM:606215", source="OMIM:606215"} ! atrioventricular septal defect
+is_a: MONDO:0020573 {source="OMIM:606215"} ! inherited disease susceptibility
+relationship: excluded_subClassOf MONDO:0020290 {source="DC-OMIM:606215", source="OMIM:606215"} ! atrioventricular septal defect
+relationship: predisposes_towards MONDO:0020290 {source="OMIM:606215"} ! atrioventricular septal defect
```

Comment:
@sabrinatoro FYI, i added this

Reply:
thank you! I thought about it, then obviously forgot! :-)

Reply:
All good! that's why we have reviewers :)